Snail/Snail+Combine/Observable+Combine.swift

```diff
@@ -0,0 +1,34 @@
+//  Copyright © 2021 Compass. All rights reserved.
+
+#if canImport(Combine)
```

Comment:
is this needed even with the availability checks or is it redundant?

Reply:
Nope! This can be removed!